pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml

```diff
@@ -255,4 +255,50 @@ data:
       - group: "oauth.openshift.io"
         resources: ["oauthclients"]
     # catch-all rule to log all other requests with request and response payloads
-    - level: RequestResponse
\ No newline at end of file
+    - level: RequestResponse
+
+  userrequests.yaml: |
+    apiVersion: audit.k8s.io/v1beta1
+    kind: Policy
+    metadata:
+      name: UserRequests
+    # Don't generate audit events for all requests in RequestReceived stage.
+    omitStages:
+    - "RequestReceived"
+    rules:
+    # Logs request and response for oauthaccess tokens.
+    - level: RequestResponse
+      resources:
+      - group: "oauth.openshift.io"
+        resources: ["oauthaccesstokens"]
```

Comment:
If I'm reading correctly, this seems to run counter to our goals of avoiding sensitive information in audit logs.  I think RequestResponse is all metadata plus the request and response bodies.  If you do this for oauthaccesstokens, I think the secrets are exposed in the log file.

Reply:
I had grabbed this audit policy from the Jira ticket.
What change do you rec to fully suppress this?